module1.js

```diff
@@ -104,12 +116,13 @@ function question1(name1) {
     "The array contains { number of animals } animals. These are: {animal1} and {animal2} and {animal3}."
 */
 
-// function question6( animArr ) {
-//   // TODO
-
-//   return /* ??? */;
-// }
+function question6( animArr ) {
+  // TODO
 
+  for (var i=0; i<animArr.length; i++) {
+    return "The array contains" + " " + animArr.length + " " + "animals." + " These are:" + " " + animArr[0] + " " + "and" + " " + animArr[1] + " " + "and" + " " + animArr[2] + "."
+}
+}
 
```

Comment:
That's OK. You might want to look at the post I made a while back on [ES6 Template Literals](https://christopherallanperry.github.io/blog/2018/01/21/template-literals.html). They should help you shorten the line of code up a bit.

Reply:
thank you crise